### PR TITLE
Github Link Banner

### PR DESCRIPTION
**Pull request by thundar001**
Updated with a GitHub alert banner that takes the user into a new tab for the Project page. Banner can be closed as well.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8790,4 +8790,4 @@
       "dev": true
     }
   }
-}
+}
```
Comment (slaurent22) [inline package-lock.json:8793]: Please don't include these in the PR. Ideally, configure VSCode to ensure newlines are added to the end of files when you save them!

File: src/components/App.tsx [hunk 2]
```diff
@@ -47,8 +49,15 @@ export default class App extends Component<AppProps, AppState> {
         this.setState(newState);
     }
     public render(): ReactNode {
+        const { alertBannerVisible, } = this.state;
         return (
             <div id="app">
+                <div className="alert-banner" style={alertBannerVisible ? {} : { display: "none", }}>
+                    <span className="close-ab" onClick={this.handleClickCloseAlertBanner.bind(this)}>&times;</span>
+                    Interested in contributing or suggesting ideas and splits? Check out the &nbsp;
```
Comment (slaurent22) [inline src/components/App.tsx:57]: Nitpick: I _think_ what you want is
```suggestion
                    Interested in contributing or suggesting ideas and splits? Check out the&nbsp;
```

**Review by slaurent22 — CHANGES_REQUESTED** [review]
Looks great! Just the  newline thing and this looks good to merge

**Comment by thundar001** [comment]
packages reverted back to previous state so only the css and app.tsx will only pull in this commit.

**Pull request merged**